plugins/hls-explicit-fixity-plugin/src/Ide/Plugin/ExplicitFixity.hs

```diff
@@ -42,6 +43,9 @@ descriptor :: Recorder (WithPriority Log) -> PluginDescriptor IdeState
 descriptor recorder = (defaultPluginDescriptor pluginId)
     { pluginRules = fixityRule recorder
     , pluginHandlers = mkPluginHandler STextDocumentHover hover
+    -- Make this plugin has a lower priority than ghcide's plugin to ensure
+    -- type info display first.
```

Comment:
This is identifying a tricky interaction with `combineResponses`: since responses are combined in the same order that plugin handlers are called, having a higher priority means *both* that you get called first, *and* that your response appears first when we combine responses. We should document this, it's important!

Indeed, do we actually care about the order in which handlers are called, or is really only the combination of responses that we care about? Perhaps we should rather be having plugin handlers return a response + a priority, rather than associating it with the plugin? (thinking out loud here)

Reply:
The ghcide-core plugin must run last'ish as it restarts the current build. Plug-ins can have side effects, so order is going to matter. This is not a nice pattern, so it would be good to offer pure alternatives. If anyone wants to contribute a better way to combine responses, I would be supportive.